ch341a_spi.c

```diff
@@ -444,7 +444,11 @@ int ch341a_spi_init(void)
 		return -1;
 	}
 
-	libusb_set_debug(NULL, 3); // Enable information, warning and error messages (only).
+#if LIBUSB_API_VERSION >= 0x01000106
+   	libusb_set_option(NULL, LIBUSB_OPTION_LOG_LEVEL, LIBUSB_LOG_LEVEL_INFO);
```

Comment:
@Asiderr Have you confirmed that LOG_LEVEL_INFO corresponds to the old debug level of 3?

Reply:
@macpijan Yes,  the LOG_LEVEL_INFO is equal 3. (http://libusb.sourceforge.net/api-1.0/group__libusb__lib.html lib is included in the file)